src/main/java/LoginTest.java

```diff
@@ -35,12 +35,46 @@ void loginTest() throws InterruptedException {
                 .typePassword(password)
                 .clickLoginButton();
 
-        HomePage homePage = loginPage.clickLoginButton();
+        homePage = loginPage.clickLoginButton();
 
         homePage
                 .getHomepageMarker()
                 .shouldBe(visible);
+    }
+
+    @Test //( enabled=false )
+    public void configTest() {
+
+        ConfigPage configPage = PageFactory.openConfigPage();
+
+        configPage
+                .getConfigPageMarker()
+                .shouldBe(visible);
+
+        configPage.inputSaveConfigValue(ConfigPage.configTitleFieldTest);
+        configPage.inputSaveConfigValue(ConfigPage.configTitleFieldDefault);
+    }
+
+    @Test //(priority=1)
+    public void adminCreateTest() {
+
+        AdminCreatePage adminCreatePage = PageFactory.openAdminCreatePage();
+        adminCreatePage.adminCreateFillForm();
+
+        try {
+            Thread.sleep(3000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
```

Comment:
1.even if we are sure we need sleep here - let's encapsulate it in some class. 
for example, create SysUtils and create static method sleep() in it that will handle exception. otherwise it's messy

2.whole logic looks weird
-we open one page and create some form
-we sleep
-we open another page

is the app supposed to work like this? are you sure there's no button to go to AdminTablePage and we need to do it through adress bar?